package.js

```diff
@@ -0,0 +1,14 @@
+// package metadata file for Meteor.js
+
+Package.describe({
+  name: 'underscorestring:underscore.string',
+  summary: 'underscore.string (official): String manipulation extensions for Underscore.js javascript library.',
+  version: '3.0.3',
+  git: 'https://github.com/epeli/underscore.string.git',
+  documentation: 'README.markdown'
+});
+
+Package.onUse(function (api) {
+  api.versionsFrom('METEOR@1.0');
+  api.addFiles('dist/underscore.string.js');
```

Comment:
Since Meteor is going to bundle and minify/uglify everithing in a unique file in any case, we ususally prefer the original source.

This could permit do debug code in a easier way...